OpenRA.Mods.Common/Traits/SupportPowers/SupportPower.cs

```diff
@@ -73,7 +73,21 @@ public abstract class SupportPowerInfo : PausableConditionalTraitInfo
 
 		public readonly string OrderName;
 
-		public SupportPowerInfo() { OrderName = GetType().Name + "Order"; }
+		// Added NumberOfTargetsOrSteps for AI Multiple SupportPower targeting and future N Target Supports
+		// ~~StormWing0 made this for AI usage mainly but hopefully at some point it can be used for more than that.
```

Comment:
Its not important who added it..... remove this line please.

Reply:
Almost couldn't tell what line you were talking about there for a moment.  Also if there's a way to get around needing the NumberOfTargetsOrSteps line itself please do hit me over the head with ideas because that's the only thing I could think of at the time that'd get the effect I was looking for while still leaving the door open for other things down the road.